README.md

```diff
@@ -82,3 +82,11 @@ Bug reports and pull requests are welcome on GitHub at https://github.com/Fusion
 
 This code is available as open source under the terms of the [Apache v2.0 License](https://opensource.org/licenses/Apache-2.0).
 
+
+## Upgrade Policy
+
+This library is built automatically to keep track of the FusionAuth API, and may also receive updates with bug fixes, security patches, tests, code samples, or documentation changes.
+
+These releases may also update dependencies, language engines, and operating systems, as we\'ll follow the deprecation and sunsetting policies of the underlying technologies that it uses.
```

Comment:
> .. systems, as we'll follow ...

This is confusing. Not sure what it means. Does it mean "because we will ..." ?  or "... per the following policies" ?